src/crimson/os/cyan_store.cc

```diff
@@ -0,0 +1,31 @@
+#include "cyan_store.h"
+
+#include <fmt/format.h>
```

Comment:
Had a compile error in my local build:
```
/root/ceph/src/crimson/os/cyan_store.cc:3:10: fatal error: fmt/format.h: No such file or directory
 #include <fmt/format.h>
          ^~~~~~~~~~~~~~
compilation terminated.
```

Reply:
@cyx1231st you might want to update the fmt submodule in seastar?

Reply:
I checked my /src/seastar HEAD is at 12b1272, and /src/seastar/fmt is at 3e75ad9

And from "make check" logs, seems it doesn't build crimson-os or crimson-osd targets...

Reply:
fixed and repushed.